Source/JavaScriptCore/b3/B3LowerMacros.cpp

```diff
@@ -287,11 +287,9 @@ class LowerMacros {
                 Vector<SwitchCase> cases;
                 for (SwitchCase switchCase : switchValue->cases(m_block))
                     cases.append(switchCase);
-                std::sort(
-                    cases.begin(), cases.end(),
-                    [] (const SwitchCase& left, const SwitchCase& right) {
-                        return left.caseValue() < right.caseValue();
-                    });
+                std::ranges::sort(cases, [] (const SwitchCase& left, const SwitchCase& right) {
```

Comment:
Please remove the space between the `[]` and the parameter list. Also can probably use auto here.

```suggestion
                std::ranges::sort(cases, [](auto& left, auto& right) {
```